shark_turbine/kernel/_support/tracing.py

```diff
@@ -121,6 +121,10 @@ class CapturedTrace:
     def __init__(self, region_graph: RegionGraph, root_graph: str):
         self.region_graph = region_graph
         self.root_graph = root_graph
+        self.region_graph.subgraphs[root_graph].subgraphs = {}
+        for name, subgraph in self.region_graph.subgraphs.items():
+            if name != root_graph:
+                self.region_graph.subgraphs[root_graph].subgraphs[name] = subgraph
```

Comment:
is it not already structured this way? otherwise why would we be able to do `for name, subgraph in self.region_graph.subgraphs.items():` ?

Reply:
ah nvm I see you are moving from `self.region_graph.subgraphs` into `self.region_graph.subgraphs[root_graph].subgraphs`. Seems useful for making the other subgraphs under the root_graph in terms of hierarchy, but not 100% sure where it's used